src/common/Exceptions/Pipeline/CMakeLists.txt

```diff
@@ -4,36 +4,72 @@
 
 #****************************************************************************
 # Modifications:
+#   Kathleen Biagas, Thu May 16, 2024
+#   Using blt, so create lists for sources, headers, includes.
 #
 #****************************************************************************
 
-target_sources(visitcommon PRIVATE
-    AbortException.C
-    BadCellException.C
-    BadDomainException.C
-    BadIndexException.C
-    BadNodeException.C
-    BadVectorException.C
-    ExpressionException.C
-    GhostCellException.C
-    ImproperUseException.C
-    IncompatibleDomainListsException.C
-    IntervalTreeNotCalculatedException.C
-    InvalidCategoryException.C
-    InvalidCellTypeException.C
-    InvalidDimensionsException.C
-    InvalidLimitsException.C
-    InvalidMergeException.C
-    InvalidSetException.C
-    InvalidConversionException.C
-    LogicalIndexException.C
-    NoCurveException.C
-    NoDefaultVariableException.C
-    NoInputException.C
-    NonQueryableInputException.C
-    QueryArgumentException.C
-    StubReferencedException.C
-    UnexpectedValueException.C)
+visit_append_list(
```

Comment:
So, is `visit_append_list()` primarily concerned with building a list of targets that span multiple different SUBdirectories? If so, if feels like the name of the method doesn't truly capture is real purpose.

Reply:
`visit_append_list` is a function that takes items and appends them to the list of the given NAME.
It has nothing to do with building anything, in and of itself.
The created lists are needed because the `visit_add_library` call  (which calls `blt_add_library`) requires that SOURCES be defined for the library (`blt` requirement, not CMake). So subdirs fill in all the lists needed by the parent's `visit_add_library` call.

The round-about way of using the function instead of directly adding things to a list created by the parent dir, is due to CMake's variable scoping rules.